SampleApps/FRExample/FRExample/ViewController.swift

```diff
@@ -162,7 +162,7 @@ class ViewController: UIViewController {
         FRURLProtocol.tokenManagementPolicy = policy
         
         //  - MARK: Authorization Policy - Example
-        let authPolicy = AuthorizationPolicy(validatingURL: [URL(string: "http://localhost:9888/policy/transfer")!], delegate: self)
+        let authPolicy = AuthorizationPolicy(validatingURL: [URL(string: "https://openig.petrov.ca/products")!], delegate: self)
```

Comment:
should we revert this URL change?

Reply:
yea, thats the production one. Good call